src/m365/spo/commands/eventreceiver/eventreceiver-get.ts

```diff
@@ -0,0 +1,157 @@
+    telemetryProps.listUrl = typeof args.options.listUrl !== 'undefined';
+    telemetryProps.scope = typeof args.options.scope !== 'undefined';
+    telemetryProps.id = typeof args.options.id !== 'undefined';
+    telemetryProps.name = typeof args.options.name !== 'undefined';
+    return telemetryProps;
+  }
+
+  public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
+    let requestUrl = `${args.options.webUrl}/_api/`;
+    let listUrl: string = '';
+    let filter: string = '?$filter=';
+
+    if (args.options.listId) {
+      listUrl = `lists(guid'${encodeURIComponent(args.options.listId)}')/`;
+    }
+    else if (args.options.listTitle) {
+      listUrl = `lists/getByTitle('${encodeURIComponent(args.options.listTitle)}')/`;
+    }
+    else if (args.options.listUrl) {
+      const listServerRelativeUrl: string = urlUtil.getServerRelativePath(args.options.webUrl, args.options.listUrl);
+      listUrl = `GetList('${encodeURIComponent(listServerRelativeUrl)}')/`;
+    }
+
+    if (!args.options.scope || args.options.scope === 'web') {
+      requestUrl += `web/${listUrl}eventreceivers`;
+    } 
+    else {
+      requestUrl += 'site/eventreceivers';
+    }
+
+    if (args.options.id) {
+      filter += `receiverid eq (guid'${args.options.id}')`;
+    } 
+    else {
+      filter += `receivername eq '${args.options.name}'`; 
+    }
+
+    const requestOptions: any = {
+      url: requestUrl + filter,
+      headers: {
+        'accept': 'application/json;odata=nometadata'
+      },
+      responseType: 'json'
+    };
+
+    request
+      .get<{ value: any[] }>(requestOptions)
+      .then((res: any): void => {
+        logger.log(res.value);
+        cb();
+      }, (err: any): void => this.handleRejectedODataJsonPromise(err, logger, cb));
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '-u, --webUrl <webUrl>'
+      },
+      {
+        option: '--listTitle [listTitle]'
+      },
+      {
+        option: '--listId  [listId]'
+      },
+      {
+        option: '--listUrl [listUrl]'
+      },
+      {
+        option: '-n, --name [name]'
+      },
+      {
+        option: '-i, --id [id]'
+      },
+      {
+        option: '-s, --scope [scope]',
+        autocomplete: ['web', 'site']
+      }
+    ];
+
+    const parentOptions: CommandOption[] = super.options();
+    return options.concat(parentOptions);
+  }
+
+  public validate(args: CommandArgs): boolean | string {
+    const isValidSharePointUrl: boolean | string = validation.isValidSharePointUrl(args.options.webUrl);
+    if (isValidSharePointUrl !== true) {
+      return isValidSharePointUrl;
+    }
+
+    const erOptions: any[] = [args.options.name, args.options.id];
+    if (erOptions.filter(item => item !== undefined).length === 0) {
+      return `Specify atleast the name or the id of the event receiver to retrieve`;
+    }
```

Comment:
for this kind of validation were we should specify at least (and no more than) one param I think we should `optionSets`
```
 public optionSets(): string[][] | undefined {
    return [
      ['name', 'id']
    ];
  }
```